pkg/types/defaults.go

```diff
@@ -10,9 +10,10 @@ const (
 
 var (
 	NcpSections      = []string{"DEFAULT", "ha", "k8s", "coe", "nsx_v3", "vc"}
-	AgentSections    = []string{"DEFAULT", "k8s", "coe", "nsx_node_agent", "nsx_kube_proxy"}
+	AgentSections    = []string{"DEFAULT", "nsx_node_agent", "nsx_kube_proxy"}
 	OperatorSections = []string{"DEFAULT", "ha", "k8s", "coe", "nsx_v3", "vc", "nsx_node_agent", "nsx_kube_proxy"}
 	BootstrapOptions = []string{"DEFAULT", "nsx_node_agent"}
```

Comment:
Do we need to recheck `BootstrapOptions`, does it have impact on node networking?

Reply:
Good point, we'd better have a check. Code updated.